File: ComissionRateApi/Entities/Order.cs
namespace ComissionRateApi.Entities;

public class Order
{
    public int Id { get; set; }
    public DateTime? OrderDate { get; set; }
    public DateTime? RequiredDate { get; set; }
    public string ShipName { get; set; }
    public string ShipAddress { get; set; }
    public string ShipCity { get; set; }
    public string ShipRegion { get; set; }
    public string ShipPostalCode { get; set; }
    public string ShipCountry { get; set; }
    public Customer Customer { get; set; }
    public int CustomerId { get; set; }
}

File: comissionRate-ui/src/app/_models/order.ts
export interface Order {
  id: number;
  orderDate: Date;
  requiredDate: Date;
  shipName: string;
  shipAddress: string;
  shipCity: string;
  shipRegion: string;
  shipPostalCode: string;
  shipCountry: string;
  customerId: number;
  customerName: string;
}
